### PR TITLE
README fixes

### PR DESCRIPTION
**Pull request by paulromano**

### CODE REVIEW

**Comment by paulromano** [comment]
One more thing: The shebang line on plot_explorer.py should be `#!/usr/bin/env python3` rather than `#!/usr/bin/python3` in case someone (like myself) is using a non-system-installed Python interpreter. I've gone ahead and fixed that here.

File: README.md [hunk 1]
```diff
@@ -1,24 +1,19 @@
-OpenMC Plot Explorer
-====================
+# OpenMC Plot Explorer
```
Comment (pshriwise) [inline README.md]: Are these header designations generally preferred over `===` or `----`?

File: README.md [hunk 2]
```diff
@@ -1,24 +1,19 @@
...
 
-Description
------------
+## Description
 
 A graphical interface for visualizing and manipulating OpenMC plot slices.
 
-Originally created by @landonjmitchel at MIT ([original
+Originally created by @landonjmitchell at the University of Chicago ([original
```
Comment (pshriwise) [inline README.md]: Thanks!!

File: README.md (hunk 1 above)
Comment (paulromano) [reply README.md]: For markdown, yes. The fact that underlines work is probably just a convenience for people converting from other markup formats (e.g. rst)

**Pull request merged**